src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+	import Img from '$lib/components/ui/Img.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { DApp } from '$lib/types/dapp';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+
+	export let dApp: DApp;
+</script>
+
+<article class="relative flex h-64 items-end overflow-hidden rounded-2xl">
+	{#if dApp.screenshots && dApp.screenshots.length > 0}
+		<div class="absolute">
+			<Img src={dApp.screenshots[0]} alt={dApp.name} />
+		</div>
+	{/if}
+	<div class="flex-1 px-4 py-4 backdrop-blur-sm">
+		<div class="flex items-center gap-x-2">
+			<div class="h-12 w-12 rounded-full">
+				<Img src={dApp.logo} alt={dApp.name} />
+			</div>
+			<div class="flex-1">
+				<h6 class="text-sm font-bold">{$i18n.dapps.text.featured}</h6>
+				<h4 class="text-white">{dApp.name}</h4>
+			</div>
+
+			<a
```

Comment:
@AntonioVentilii-DFINITY do we have already a maybe "Button link" component something that can potentially be used here?

Reply:
Correct! We have `ExternalLinkAsIcon` that is a derivative of Button

Reply:
I think we can resolve this issue, I clarified with artem, this button will not directly navigate. Instead it will open the modal. Will change it to a button